lib/util/state.js

```diff
@@ -18,6 +18,7 @@ const defaultOptions = {
     ignoreDev: false,
     forceColor: false,
     saveExact: false,
+    installer: 'npm',
```

Comment:
ah, yes, this is correct! just use `currentState.get('installer')` to get the value. :smile: